components/Today.js

```diff
@@ -19,22 +35,32 @@ function Today({ today, dispatch }) {
                    onPress={() => dispatch({type: 'DECREMENT'})}
         >
           <Icon name="back-in-time" type="entypo" color="white" size={24} style={{
-              borderStyle: 'solid', borderRadius: 30, borderWidth: 1, borderColor: 'white', padding: 12,
-              marginRight: 10,
-          }} />
+            borderStyle: 'solid',
+            borderRadius: '100%',
+            borderWidth: 1,
+            borderColor: 'rgba(255, 255, 255, .5)',
+            padding: 12,
+            marginRight: 10,
+          }}/>
           <Text style={{fontWeight: "600"}}>Previous Workout</Text>
         </Pressable>
-            <Text style={styles.title}>
-              This is workout #{today}:
-            </Text>
-            <TodaysWorkout />
-      </ScrollView>
-    </KeyboardAvoidingView>
+      </View>
+
+      <KeyboardAvoidingView behavior="padding" style={{width: "100%"}}>
```

Comment:
I'm curious to come back to this later to confirm that it's all gravy, but mostly to better understand the significance/use/implications of the KeyboardAvoidingView & ScrollView components.